units/modprobe@.service

```diff
@@ -33,3 +33,4 @@ StartLimitIntervalSec=0
 [Service]
```

Comment:
Maybe better to replace Before=sysinit.target with modules-load.target?

Reply:
Definitely.